glue/flagging_script_glue/flagging_e894e5.py

```diff
@@ -884,6 +884,17 @@ def get_id(row: pd.Series, col: str) -> str:
     column = col + "_name"
     words = str(row[column]).lower()
 
+    # Check for missing values first
+    if pd.isnull(row[column]) or words in [
+        "none",
+        "nan",
+        "unknown",
+        "missing seller name",
+        "missing buyer name",
+    ]:
+        id = "Empty Name"
+        return id
+
```

Comment:
The prior flagging script was erroneously matching values with different string representations of NA. Since there are null values, the `words = str(row[column]).lower()` directly above this code chunk was converting those to `none` or `nan`, which returned family sales matches. 

There were also other already present values such as `UNKOWN`, `MISSING SELLER NAME`, and `MISSING BUYER NAME`, which were returning matches. We decide here to manually change these values to `Empty Name`, using the existing strategy.

Reply:
**praise:** Nice catch @wagnerlmichael.

**Question:** I assume that the name matching down the line discards any matches on `"Empty Name"`, is that correct?

Reply:
Yes, that is correct